gits/terminal.py

```diff
@@ -342,10 +342,13 @@ def _cap_dim(self):
         self._cap_set_color(2)
 
     def _cap_dl(self, n):
-        """Deletes ``n`` number of lines. """
+        """Deletes ``n`` number of lines in their entirety, with the lines
+        below moving up to fill the gap that is left.
+        """
```

Comment:
From the documentation related to AIX.

> `dl1` deletes the line the cursor is on. This is done only from the first position on the line to be deleted.

The only difference between `dl1` and `dl` is the latter accepts a number of lines to be deleted. But from the description I could see that to delete a line via `dl1` I have to set `x` to 0 and `y` to the number of the target line.